modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/mapper/JsonObjectMapper.java

```diff
@@ -1,6 +1,6 @@
 /********************************************************************************
- * Copyright (c) 2022, 2023 T-Systems International GmbH
- * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022, 2023, 2024 T-Systems International GmbH
```

Comment:
This copyright header is incorrect again, please fix it as described in previous PRs.

Reply:
Updated all file copyright